daemons/controld/controld_te_events.c

```diff
@@ -327,7 +327,7 @@ get_cancel_action(const char *id, const char *node)
 
             task = crm_element_value(action->xml, PCMK__XA_OPERATION_KEY);
             if (!pcmk__str_eq(task, id, pcmk__str_casei)) {
-                crm_trace("Wrong key %s for %s on %s", task, id, node);
+                crm_trace("Wrong key %s for %s on %s", task, id, node ? node : "(null)");
```

Comment:
FYI we have a convenience macro for that ternary, used like `pcmk__s(node, "(null)")`

But in this case I think the log is useless, so we can just drop it